animation_nodes/nodes/gpencil/gp_stroke_from_points.py

```diff
@@ -0,0 +1,56 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+from ... data_structures import GPStroke, VirtualDoubleList
+
+class GPStrokeFromPointsNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPStrokeFromPointsNode"
+    bl_label = "GP Stroke From Points"
+    errorHandlingType = "EXCEPTION"
+
+    useStrengthsList: VectorizedSocket.newProperty()
+    usePressuresList: VectorizedSocket.newProperty()
+    useUVRotationsList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("Vector List", "Points", "vertices")
+        self.newInput(VectorizedSocket("Float", "useStrengthsList",
+            ("Strength", "strength"), ("Strengths", "strengths")), value = 1)
+        self.newInput(VectorizedSocket("Float", "usePressuresList",
+            ("Pressure", "pressure"), ("Pressures", "pressures")), value = 1)
+        self.newInput(VectorizedSocket("Float", "useUVRotationsList",
+            ("UV-Rotation", "uvRotation"), ("UV-Rotations", "uvRotations")), hide = True)
+        self.newInput("Float", "Line Width", "lineWidth", value = 250)
+        self.newInput("Boolean", "Cyclic", "drawCyclic", value = False, hide = True)
+        self.newInput("Text", "Start Cap Mode", "startCapMode", value = 'ROUND', hide = True)
+        self.newInput("Text", "End Cap Mode", "endCapMode", value = 'ROUND', hide = True)
+        self.newInput("Integer", "Material Index", "materialIndex", value = 0)
+        self.newInput("Text", "Display Mode", "displayMode", value = 'SCREEN', hide = True)
+        self.newOutput("GPStroke", "Stroke", "stroke")
+
+    def execute(self, vertices, strengths, pressures, uvRotations, lineWidth, drawCyclic,
+                startCapMode, endCapMode, materialIndex, displayMode):
+
+        amount = len(vertices)
+        stroke = GPStroke()
```

Comment:
Move the everything before this statement and pass them to the constructor.

Reply:
Replaced.